src/main.c

```diff
@@ -277,7 +482,46 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
         return;
     }
 
-    Iface = GetInterface(LowerIfIndex);
+    // VMSwitchPAcketFragment -> VMSwitch traffic
+    if (ev->EventHeader.EventDescriptor.Id == tidVMSwitchPacketFragment) {
+        //Get VLAN from OOB
+        VMSwitchPacketFragment = malloc(sizeof(VMSWITCH_PACKET_FRAGMENT));
```

Comment:
Don't do a malloc here. The etl2pcapng code has been carefully written to avoid malloc calls in the packet callback for performance.

Reply:
fixed